package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "uswds",
-  "version": "2.11.2",
+  "version": "2.14.0-beta.1",
```

Comment:
Used `2.14.0-beta.1` here because it wouldn't pass with something like `library-beta.1` — it's just as accurate as `2.10.0-beta.0` I guess